dependency_manager/src/edm_tool/bazel.py

```diff
@@ -0,0 +1,55 @@
+"Bazel related functions for edm_tool."
+import sys
+
+import yaml
+
+
+def generate_deps(args):
+    "Parse the dependencies.yaml and print content of *.bzl file to stdout."
+    with open(args.dependencies_yaml, 'r', encoding='utf-8') as f:
+        deps = yaml.safe_load(f)
+    print("Buid files: ", args.build_file, file=sys.stderr)
+    if args.build_file:
+        build_files = dict((f.split(":")[1], f) for f in args.build_file)
```

Comment:
if the argument is called `build_file`, how can you create multiple `build_files` out of it?  Maybe only a naming issue.  Would be good to have a help string for the argument in edm to be able to reason about it.

Reply:
This option can be supplied multiple times.
And every value ist a bazel label `@workspace//path/to/build/file:BUILD.<depaname>.bazel`

We are converting a list of such strings into a dict:
```python
{
  'BAZEL.<depname>.bazel': `@workspace//path/to/build/file:BUILD.<depaname>.bazel`,
}
```

Reply:
I've refactored it a bit.
Now it should be way cleaner as well.